tests-perf/ops/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.9-alpine3.13
+
+ENV PYTHONDONTWRITEBYTECODE 1
+
+RUN apk add --no-cache bash build-base git libtool cmake autoconf automake gcc musl-dev postgresql-dev g++ libexecinfo-dev make libffi-dev libmagic libcurl curl-dev rust cargo && rm -rf /var/cache/apk/*
+
+# update pip
+RUN python -m pip install wheel
+RUN python -m pip install --upgrade pip
+
+RUN set -ex && mkdir /app
+
+WORKDIR /app
+
+COPY . /app
+
+RUN set -ex && pip3 install -r requirements_for_test.txt
```

Comment:
`set -ex` provides no value here and can be removed.

gnu.org/software/bash/manual/html_node/The-Set-Builtin.html